src/ram/AbstractConditional.h

```diff
@@ -64,6 +58,12 @@ class AbstractConditional : public NestedOperation {
         return NestedOperation::equal(node) && equal_ptr(condition, other.condition);
     }
 
+    NodeVec getChildNodesImpl() const override {
```

Comment:
Can we rename getChildNodesImpl to something more readable? For example, getChildren() etc.

Reply:
This matches `ast::Node`. Do we want to rename the one over there too?